resolwe_bio/tools/make_igv_session_archive.py

```diff
@@ -13,10 +13,31 @@
 args = parser.parse_args()
 
 
-def make_xml_tree(input_file):
+def get_build_info(input_file):
+    """Return build information in UCSC version or empty string."""
+
+    build_dict = {
+        'hg38': 'GRCh38',
+        'hg19': 'GRCh37',
```

Comment:
Dictionaries must not have two identical keys (`hg19`). The second value will overrride the first one. Check what happens if you define a dict in this way: `d = {'a':1, 'a':2}`. You get `d = {'a':2}`.